test/parallel/test-cli-node-options.js

```diff
@@ -24,6 +24,13 @@ expect('--throw-deprecation', 'B\n');
 expect('--zero-fill-buffers', 'B\n');
 expect('--v8-pool-size=10', 'B\n');
 expect('--trace-event-categories node', 'B\n');
+expect('--perf-basic-prof', 'B\n');
+
+if (common.isLinux) {
+  // PerfJitLogger is only implemented in Linux.
+  // https://github.com/nodejs/node/pull/17600#issuecomment-350664113
```

Comment:
I think it's safe to drop the GitHub link.

Reply:
@cjihrig Thank you for review.
I got it. I removed Github link.

Reply:
@Leko FWIW this is written in the comments in `deps/v8/src/perf-jit.h`